Domains/4-Application/Connectors/CivilInfrastructureFramework/CifRail.ecschema.xml

```diff
@@ -818,7 +823,15 @@
         <ECProperty propertyName="Length" typeName="double" displayLabel="Length" category="Joint_Joint" priority="299930" kindOfQuantity="cifu:LENGTH"/>
         <ECProperty propertyName="Station" typeName="double" displayLabel="Station" category="Joint_Joint" priority="299920" kindOfQuantity="cifu:STATION"/>
         <ECProperty propertyName="Offset" typeName="double" displayLabel="Offset" category="Joint_Joint" priority="299940" kindOfQuantity="cifu:LENGTH"/>
-        <ECProperty propertyName="Rail" typeName="double" displayLabel="Rail" category="Joint_Joint" priority="299910" kindOfQuantity="cifu:LENGTH"/>
+        <ECProperty propertyName="RailHand" typeName="string" displayLabel="Rail" category="Joint_Joint" priority="299910"/>
+        <ECProperty propertyName="Rail" typeName="double" category="Joint_Joint" priority="299910" kindOfQuantity="cifu:LENGTH">
+			<ECCustomAttributes>
+                <HiddenProperty xmlns="CoreCustomAttributes.01.00.04"/>
+                <Deprecated xmlns="CoreCustomAttributes.01.00.04">
+                    <Description>Rail is not a double.It was a bug in CIF</Description>
```

Comment:
The "Description" of the Deprecated CA is aimed at an existing consumer of a schema. Thus, rather than explaining why a property is being deprecated, it should instead advertise what an existing consumer needs to do in order to read the same data going forward. In this case, the description could be something like "Use RailJointAspect:RailHand instead".

Reply:
Updated the description